scripts/verify-mcp-tools.sh

```
@@ -0,0 +1,417 @@
+  echo -e "\n${BLUE}═══════════════════════════════════════════════════════${NC}"
+  echo -e "${BLUE}  $1${NC}"
+  echo -e "${BLUE}═══════════════════════════════════════════════════════${NC}"
+}
+
+# Helper function to call MCP tool
+call_mcp_tool() {
+  local tool_name=$1
+  local arguments=$2
+  local id=${3:-1}
+  
+  local request=$(cat <<EOF
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)